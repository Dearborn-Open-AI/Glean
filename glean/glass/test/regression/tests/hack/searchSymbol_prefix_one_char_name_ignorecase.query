action: searchSymbol
args: {"name": "s", "repo_name": "test", "language": [3], "kinds": [], "options": {"detailedResults": false, "exactMatch": false, "ignoreCase": true, "namespaceSearch": false } }
