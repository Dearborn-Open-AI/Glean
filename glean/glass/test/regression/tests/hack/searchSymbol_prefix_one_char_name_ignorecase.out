[
    "@generated",
    {
        "symbolDetails": [],
        "symbols": [
            {
                "kind": 7,
                "language": 3,
                "location": {
                    "filepath": "www/SourceClass.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 13,
                        "lineEnd": 38
                    },
                    "repository": "test"
                },
                "name": "SourceClass",
                "qname": {
                    "container": "",
                    "localName": "SourceClass"
                },
                "score": {
                    "exactness": 2
                },
                "symbol": "test/php/SourceClass"
            },
            {
                "kind": 12,
                "language": 3,
                "location": {
                    "filepath": "www/SourceEnum.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 10,
                        "lineEnd": 13
                    },
                    "repository": "test"
                },
                "name": "SourceEnum",
                "qname": {
                    "container": "",
                    "localName": "SourceEnum"
                },
                "score": {
                    "exactness": 2
                },
                "symbol": "test/php/SourceEnum"
            },
            {
                "kind": 12,
                "language": 3,
                "location": {
                    "filepath": "www/SourceEnum.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 15,
                        "lineEnd": 19
                    },
                    "repository": "test"
                },
                "name": "SourceEnum2",
                "qname": {
                    "container": "",
                    "localName": "SourceEnum2"
                },
                "score": {
                    "exactness": 2
                },
                "symbol": "test/php/SourceEnum2"
            },
            {
                "kind": 12,
                "language": 3,
                "location": {
                    "filepath": "www/SourceEnum.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 44,
                        "lineBegin": 23,
                        "lineEnd": 23
                    },
                    "repository": "test"
                },
                "name": "SourceEnumClass3",
                "qname": {
                    "container": "",
                    "localName": "SourceEnumClass3"
                },
                "score": {
                    "exactness": 2
                },
                "symbol": "test/php/SourceEnumClass3"
            },
            {
                "kind": 13,
                "language": 3,
                "location": {
                    "filepath": "www/SourceInterface.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 10,
                        "lineEnd": 18
                    },
                    "repository": "test"
                },
                "name": "SourceInterface",
                "qname": {
                    "container": "",
                    "localName": "SourceInterface"
                },
                "score": {
                    "exactness": 2
                },
                "symbol": "test/php/SourceInterface"
            },
            {
                "kind": 31,
                "language": 3,
                "location": {
                    "filepath": "www/SourceTrait.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 10,
                        "lineEnd": 18
                    },
                    "repository": "test"
                },
                "name": "SourceTrait",
                "qname": {
                    "container": "",
                    "localName": "SourceTrait"
                },
                "score": {
                    "exactness": 2
                },
                "symbol": "test/php/SourceTrait"
            },
            {
                "context": {
                    "qname": {
                        "container": "",
                        "localName": "SourceTrait"
                    },
                    "symbol": "test/php/SourceTrait"
                },
                "kind": 8,
                "language": 3,
                "location": {
                    "filepath": "www/SourceTrait.php",
                    "range": {
                        "columnBegin": 3,
                        "columnEnd": 4,
                        "lineBegin": 15,
                        "lineEnd": 17
                    },
                    "repository": "test"
                },
                "name": "superAdd",
                "qname": {
                    "container": "SourceTrait",
                    "localName": "superAdd"
                },
                "score": {
                    "exactness": 2
                },
                "symbol": "test/php/SourceTrait/superAdd"
            },
            {
                "kind": 7,
                "language": 3,
                "location": {
                    "filepath": "www/SourceClass.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 40,
                        "lineEnd": 45
                    },
                    "repository": "test"
                },
                "name": "SubClass",
                "qname": {
                    "container": "",
                    "localName": "SubClass"
                },
                "score": {
                    "exactness": 2
                },
                "symbol": "test/php/SubClass"
            },
            {
                "kind": 7,
                "language": 3,
                "location": {
                    "filepath": "www/SuperClass.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 10,
                        "lineEnd": 21
                    },
                    "repository": "test"
                },
                "name": "SuperClass",
                "qname": {
                    "container": "",
                    "localName": "SuperClass"
                },
                "score": {
                    "exactness": 2
                },
                "symbol": "test/php/SuperClass"
            },
            {
                "context": {
                    "qname": {
                        "container": "",
                        "localName": "SuperClass"
                    },
                    "symbol": "test/php/SuperClass"
                },
                "kind": 8,
                "language": 3,
                "location": {
                    "filepath": "www/SuperClass.php",
                    "range": {
                        "columnBegin": 3,
                        "columnEnd": 4,
                        "lineBegin": 14,
                        "lineEnd": 16
                    },
                    "repository": "test"
                },
                "name": "superAdd",
                "qname": {
                    "container": "SuperClass",
                    "localName": "superAdd"
                },
                "score": {
                    "exactness": 2
                },
                "symbol": "test/php/SuperClass/superAdd"
            }
        ]
    }
]